src/webvr-manager.js

```diff
@@ -104,10 +106,12 @@ function WebVRManager(renderer, effect, params) {
       default:
         this.setMode_(Modes.NORMAL);
     }
-    this.button.on('fs', this.onFSClick_.bind(this));
-    this.button.on('vr', this.onVRClick_.bind(this));
-    this.button.on('back', this.onBackClick_.bind(this));
-    this.button.on('settings', this.onSettingsClick_.bind(this));
+    if (this.button) {
```

Comment:
If you don't want to support options for changing or disabling the buttons, I agree with you.